manifest.webapp

```diff
@@ -13,6 +13,9 @@
   "default_locale": "en",
   "orientation": ["portrait-primary"],
   "permissions": {
+    "geolocation": {
+       "description": "Required by location API."
+    },
```

Comment:
In a followup, we should start preprocessing this file as well, so we only request the "geolocation" permission if the Java API is enabled.